### ISSUE TITLE
[Bug]: Latest Automatic update breaks openOutpaint

### ISSUE THREAD

**Issue by vladmandic** [zero01101/openOutpaint-webUI-extension#31]
### Is ths issue about the extension?

- [X] Yup, this is for sure about the extension

### What happened?

Updated Automatic1111 to latest (hash #6faae2323963f9b0e0086a85b9d0472a24fbaa73)
openOutpaint Extension stops working:
- Browser console reports: `Failed to load resource: the server responded with a status of 500 (Internal Server Error)`
- Automatic WebUI console reports Gradio errors on each poll from openOutpaint (every few seconds):

```text
pydantic.error_wrappers.ValidationError: 1 validation error for Options
response -> show_progressbar
  value is not None (type=type_error.not_none)
ERROR:    Exception in ASGI application
```

basically, `options` object now has one more mandatory field `show_progressbar` that **openOutpaint** is not sending via POST, so its raising validation errors.

### Steps to reproduce the problem

<see above>

### What should have happened?

<see above>

### Commit where the problem happens

201456ce36f2f6ec81067003199c4d77d4c57fbf

### What platforms do you use to access openOutpaint?

Windows

### What browsers do you use to access the UI ?

Microsoft Edge

### Browser Extensions/Addons

N/A

### AUTOMATIC1111 webUI Commandline Arguments

--api --xformers

**Comment by seijihariki** [comment]
Hi! I could not really replicate the error you are describing. From what I understand, we never have to actually send anything to options, and only POST to it for changing the model and for setting color correction and mask weight on boot if the user accepts it.

There was actually another problem (unrelated to show_progressbar), but seems to have been fixed by a966a1e706e9c4d65bdb63789d257626af2d6e77.

Are your problems still happening?

**Comment by vladmandic** [comment]
confirm as working now.

btw, there is a minor issue where "send to openoutpaint" button doesnt automatically switch tabs anymore  
(image does get transferred, so it more cosmetic than functiona issue)

**Comment by zero01101** [comment]
send to openOutpaint buttons should be fixed as of https://github.com/zero01101/openOutpaint-webUI-extension/commit/3ebc1812ecdfd3578852dfb09532ddc53a3728a2
